README.md

```diff
@@ -1 +1,101 @@
-# oe_migration
+# OpenEuropa oe_migration
+
+The OpenEuropa oe_migration provides extensions to the core migration framework, to migrate data from Drupal 7 to the OpenEuropa project.
```

Comment:
Same as above, no need to use the machine name.

Reply:
Thank you for your precious time @imanoleguskiza